t/run_docker.sh

```diff
@@ -5,7 +5,9 @@ dockerfail=0
 cd docker
 for t in *.sh; do
     [ -x "$t" ] || continue
-    ./$t
+    for i in $(seq 1 3); do
+        ./$t && s=0 && break || s=$? && sleep 10;
+    done
     if [ $? -eq 0 ] ; then
```

Comment:
Hm, I think `$?` will always be 0 here, and the actual return code we want is stored in `$s`, no?

Reply:
pened #193